wda/__init__.py

```diff
@@ -888,6 +889,32 @@ def swipe(self, x1, y1, x2, y2, duration=0):
 
         data = dict(fromX=x1, fromY=y1, toX=x2, toY=y2, duration=duration)
         return self._session_http.post('/wda/dragfromtoforduration', data=data)
+    
+    def swipe_with_velocity(self, x1: Union[int, float], y1: Union[int, float], x2: Union[int, float], y2: Union[int, float],
+                            press_duration: float, hold_duration: float, velocity: float):
+        """
+        Press down and drag with velocity, appium forked version of wda only
+
+        Args:
+            - x1, y1, x2, y2 (Union[int, float]): The start and end coordinates if all value is of `int` type, otherwise the start
+                and end percentage of screen axis
+            - press_duration (float): the duration before swiping
+            - hold_duration (float): the duration after swiping
+            - velocity (float): the velocity of swiping
```

Comment:
what is the range of velocity and the normal value? suggest to add to doc

Reply:
doc enriched